conda/compat.py

```diff
@@ -55,13 +51,6 @@ def lchmod(path, mode):
             if not os.path.islink(path):
                 os.chmod(path, mode)
     from itertools import izip_longest as zip_longest
-    from math import log
-    def log2(x):
-        return log(x, 2)
-    def ceil(x):
-        from math import ceil
-        return int(ceil(x))
-    from pipes import quote
```

Comment:
Are these removals no longer used in the project?  Even if, removal still concerns me because other repos `import conda`.  Use `# NOQA` if you have to.